### PR TITLE
Add Music Stuff and Animations.

### PR DESCRIPTION
**Pull request by KoolKat69**
Add Music Stuff and Animations.
added Code to auto play music on start up and randomize the songs

### CODE REVIEW

File: assets/Meow/DynamicAudio.gd
Comment (rDarge) [inline assets/Meow/DynamicAudio.gd]: it doesn't really matter that much for the purpose of the project, but I think this is a good opportunity to try making this code more efficient - right now, we have to scan all of the files in the music directory (in dir_counter) each time a new song is picked; is there a way we can scan the folders just once? 

extra credit 1: Is there a way we can avoid having to scan through the folders again when we call get_audio?

I don't want to make this a chore, so if you don't feel like refactoring the code it's nbd

File: assets/Meow/DynamicAudio.gd
Comment (rDarge) [inline assets/Meow/DynamicAudio.gd]: let's delete this if it's not used and mostly the same as the other two functions

**Review by rDarge — COMMENTED** [review]
These changes look good! It'll be nice to have some more sprites we can start using for enemies and effects. If possible, can  you finish converting the .wav files to MP3s before we merge this? Also, if you could add in a txt file with the sources for the assets you added with these changes that would be great - it will help us track down the authors if we have any questions or want to find more of their work, and we may want to include them in the credits etc.

**Comment by KoolKat69** [comment]
Updated the code for the audio. swapped the wav for mp3. Only one wav file remains as it didnt provide a mp3 can convert but left it alone for now. few functions are no longer needed but left them in there for now till i know i did the pushing to github right so u can see before and after.

**Review by rDarge — APPROVED** [review]
Changes look good to me. Let's cool down on adding more sprites till we can actually get them in game.

**Pull request merged**
